src/Features/CSharp/Portable/Completion/CompletionProviders/SnippetCompletionProvider.cs

```diff
@@ -30,6 +31,14 @@ namespace Microsoft.CodeAnalysis.CSharp.Completion.Providers
     [Shared]
     internal sealed class SnippetCompletionProvider : LSPCompletionProvider
     {
+        private static readonly HashSet<string> s_builtInSnippets = new()
```

Comment:
Are those all the built-in "old" snippets from VS? Do we have a replacement for each one of them?

Reply:
This is a complete list built-in snippets that I got from the SnippetInfoService. The idea is to replace all of these with semantic snippets but I am turning them all off even if I don't have a replacement so as to not conflate the two experiences.

Reply:
If we are going after all of them, consider filtering with their file path instead of maintain a list. For example, if you go to `[VS folder]\VC#\Snippets\1033` there seems to have more snippets.

Reply:
:+1: to what @genlu said. We control what and where these onld snippets live so we can change them to make them easier to detect if necessary